com.unity.netcode.gameobjects/Runtime/Timing/NetworkTickSystem.cs

```diff
@@ -53,6 +53,12 @@ public NetworkTickSystem(uint tickRate, double localTimeSec, double serverTimeSe
             ServerTime = new NetworkTime(tickRate, serverTimeSec);
         }
 
+        internal void Reset(double localTimeSec, double serverTimeSec)
```

Comment:
I think this should be public

Reply:
What for? I'm not sure why users would need to do this